src/router/index.js

```diff
@@ -25,11 +25,15 @@ const router = new Router({
 })
 
 const checkAuth = async (to, from, next) => {
-  const token = await localforage.getItem('token')
-  store.dispatch('setToken', { token })
-
-  if (to.name !== 'auth.index' && token === null) {
-    next({ name: 'auth.index' })
+  if (!to.matched.some(record => record.meta.requiresAuth)) {
```

Comment:
Eu não compreendi pq usar .some() aqui!

Não seria apenas verificar o `to.meta.requiresAuth` se é true ou false?

Reply:
é verdade :) acho que estava pensando em outra coisa haha. Já corrigi